adr/012-auth.md

```diff
@@ -0,0 +1,30 @@
+# 12. Authentication and Authorization
```

Comment:
suggestion to also include a sequence diagram or flow diagram with this in order to better visualize the flow that is described.

Reply:
That's a great idea.  I'll work on that.